youtube-channels.csv

```diff
@@ -546,3 +546,10 @@ UCb4XH3v5SKMQXJWL_lAiUbg, beatsolow, https://www.youtube.com/channel/UCb4XH3v5SK
 UCxT-uvIzsMlvGXDQu0omuhQ, Fern, https://www.youtube.com/channel/UCxT-uvIzsMlvGXDQu0omuhQ
 UCtTly50qDGxwwcNw3u0sx3Q, Dito Korkelia, https://www.youtube.com/channel/UCtTly50qDGxwwcNw3u0sx3Q
 UC-tzG7z4619UJIANeyYWXDg, Pasti's Library, https://www.youtube.com/channel/UC-tzG7z4619UJIANeyYWXDg
+UC6murUWtqOwnTL68pwjoGjQ, EuphoricHardStyleZ , https://www.youtube.com/user/EuphoricHardStyleZ
+UCBuN7YIHjB40uXezntguDfQ, ConnecteD {Hard Dance Music}, https://www.youtube.com/channel/UCBuN7YIHjB40uXezntguDfQ
+UC2GC6a9WIw14ZPZjWbIlqlA, TrancesMaster, https://www.youtube.com/user/TrancesMaster
```

Comment:
please remove this one as track names are weirdly formatted and we won't be able to parse them